### PR TITLE
E_CLASSROOM-342 [FE/BE] Implement Change Password for Student Profile

### PR DESCRIPTION
**Pull request by ErickDoroy**
### Issue Link
- https://framgiaph.backlog.com/view/E_CLASSROOM-342
### Definition of Done
- [x] Design is the same with the figma design
- [x] User can change their password
- [x] Error handling is present

## Related PR:
- https://github.com/framgia/sph-classroom-els-be/pull/90
#### Main note
- http://localhost:3003/profile/change-password
#### Pages Affected
- Profile View Page
- Change Password Page

### Scenarios/ Test Cases
- [x] it should present error handling
- [x] It should change the password
#### User Interface
- N/A
#### User Experience 
- N/A
#### Functionality
- [ ] when inputting the value you should expected that it will return to profile view page
- [ ] it should change the password to a new password
### Screenshots
image
image
image
studentchangepass

### CODE REVIEW

**Comment by catampongan** [comment]
## Approved PR #202 
- [x] Design is the same with the figma design
- [x] User can change their password
- [x] Error handling is present
----------------------------
task-342
task-342-2

------------------------
- Total Test Time: 0.25 hr
- Total Iterations: 0
- Total Comments: 0

File: src/pages/student/profile/ChangePassword/index.js [hunk 1]
```diff
@@ -1,27 +1,71 @@
...
-import { Link } from 'react-router-dom';
-import Button from '../../../../components/Button';
-import InputField from '../../../../components/InputField';
+import { Link, useHistory } from 'react-router-dom';
+import { useToast } from '../../../../hooks/useToast';
 import Container from 'react-bootstrap/Container';
 import Form from 'react-bootstrap/Form';
+
 import style from './index.module.scss';
+import AdminApi from '../../../../api/Admin';
```
Comment (karlolee50) [inline src/pages/student/profile/ChangePassword/index.js]: please use another API. not the admin API

File: src/pages/student/profile/ChangePassword/index.js [hunk 2]
```diff
@@ -1,27 +1,71 @@
...
+  const handleOnSubmit = async ({
+    password,
+    new_password,
+    password_confirmation
+  }) => {
+    setSubmitStatus(true);
+    toast('Processing', 'Updating your current password...');
+    setErrors({});
+
+    await AdminApi.updatePassword({
```
Comment (karlolee50) [inline src/pages/student/profile/ChangePassword/index.js]: use another API. or make another API. same with my previous comment

**Review by karlolee50 — CHANGES_REQUESTED** [review]
Review time: 5 minutes
Comment/s: 2

**Review by karlolee50 — APPROVED** [review]

**Pull request merged**